lib/proxy.js

```diff
@@ -138,8 +138,8 @@ function makeWsProxy(host, port, urlPrefix, isHttps) {
   };
 
   var logException = function(e) {
-    proxyLog.warn(toString()+' Exception caught. Message='+e.message);
-    proxyLog.warn("Stack trace follows\n"+e.stack);
+    proxyLog.warn("ZWED0041W", toString(), e.message); //proxyLog.warn(toString()+' Exception caught. Message='+e.message);
```

Comment:
Says  "ZWED0041W":"% Exception caught. Message=%s",
Should say
 "ZWED0041W":"%s Exception caught. Message=%s",

Reply:
Addressed.